managed/models/location_helpers.go

```diff
@@ -376,6 +376,7 @@ func RemoveBackupLocation(q *reform.Querier, id string, mode RemoveMode) error {
 	}
 
 	for _, a := range artifacts {
+		// TODO removing artifact this way is not correct. Should be done via calling "removal service".
```

Comment:
When it will be done?

Reply:
Let's discuss it here or bring it to the daily call. I noticed that if user force deletes location, artifact files are kept on the storage. Should we give option to remove files as well? Or should we just notify user that files won't be removed? Or some other options?

Reply:
Let's discuss it here or bring it to the daily call. I noticed that if user force deletes location, artifact files are kept on the storage. Should we give option to remove files as well? Or should we just notify user that files won't be removed? Or some other options?

Reply:
Decided to leave as is for now. Just leaving the TODO to not forget to discuss it later.